backend/vendor/assets/stylesheets/jquery.jstree/themes/spree/style.scss

```diff
@@ -0,0 +1,216 @@
+/*
+ * jsTree Spree theme 1.0
+ */
+
+.jstree-spree {
+
+  li, li ul {
+    border-color: #ededed;
+    border-style: solid;
+  }
+
+  li {
+    margin-left: 26px;
+    border-width: 0 0 1px 0;
```

Comment:
Shorthand form for property `border-width` should be written more concisely as `0 0 1px` instead of `0 0 1px 0`